modules/administration/pages/monitoring.adoc

```diff
@@ -120,14 +117,14 @@ See https://github.com/boynux/squid-exporter.
 
 == Install and Configure Prometheus
 
-Prometheus is installed from a package, and needs configuration before you can use it to gather metrics.
+Prometheus is installed on your monitoring server from a package, and needs configuration before you can use it to gather metrics.
 
 
 
 === Installing Prometheus
 
 .Procedure: Installing Prometheus
-. Install the [package]``golang-github-prometheus-prometheus`` package:
+. On the monitoring server, install the [package]``golang-github-prometheus-prometheus`` package:
 +
 ----
 zypper in golang-github-prometheus-prometheus
```

Comment:
While this is the correct manual way of installing the package when logged in to the monitoring server, we could mention here that the package could also be installed using SUSE Manager itself in case the monitoring server is a managed client. There is several ways to install a package via SUSE Manager, for example schedule the installation from the UI or API, or create a package state to make sure the package is always installed and possibly updated with every application of the *highstate*.

The same would apply for the installation of Grafana as well.